packages/redux/src/actions/content.ts

```diff
@@ -15,136 +15,28 @@
  */
 
 import { Descriptor, Item, NavigationItem } from '@craftercms/models';
+import { createAction } from "@reduxjs/toolkit";
 
-export const GET_ITEM = 'CRAFTERCMS_GET_ITEM';
-export const GET_ITEM_COMPLETE = 'CRAFTERCMS_GET_ITEM_COMPLETE';
-export const GET_DESCRIPTOR = 'CRAFTERCMS_GET_DESCRIPTOR';
-export const GET_DESCRIPTOR_COMPLETE = 'CRAFTERCMS_GET_DESCRIPTOR_COMPLETE';
-export const GET_CHILDREN = 'CRAFTERCMS_GET_CHILDREN';
-export const GET_CHILDREN_COMPLETE = 'CRAFTERCMS_GET_CHILDREN_COMPLETE';
-export const GET_TREE = 'CRAFTERCMS_GET_TREE';
-export const GET_TREE_COMPLETE = 'CRAFTERCMS_GET_TREE_COMPLETE';
+export const getItem = /*#__PURE__*/ createAction<string>('GET_ITEM');
 
-export const GET_NAV = 'CRAFTERCMS_GET_NAV';
-export const GET_NAV_COMPLETE = 'CRAFTERCMS_GET_NAV_COMPLETE';
-export const GET_NAV_BREADCRUMB = 'CRAFTERCMS_GET_NAV_BREADCRUMB';
-export const GET_NAV_BREADCRUMB_COMPLETE = 'CRAFTERCMS_GET_NAV_BREADCRUMB_COMPLETE';
+export const getItemComplete = /*#__PURE__*/ createAction<{ url: string, item?: Item }>('GET_ITEM_COMPLETE');
 
-export function getItem(itemUrl: string) {
-  return {
-    type: GET_ITEM,
-    payload: itemUrl
-  }
-}
+export const getDescriptor = /*#__PURE__*/ createAction<string>('GET_DESCRIPTOR');
 
-export function getItemComplete(itemData: {
-  url: string,
-  item?: Item
-}) {
-  return {
-    type: GET_ITEM_COMPLETE,
-    payload: itemData
-  }
-}
+export const getDescriptorComplete = /*#__PURE__*/ createAction<{ url: string, descriptor?: Descriptor }>('GET_DESCRIPTOR_COMPLETE');
 
-export function getDescriptor(url: string) {
-  return {
-    type: GET_DESCRIPTOR,
-    payload: url
-  }
-}
+export const getChildren = /*#__PURE__*/ createAction<string>('GET_CHILDREN');
 
-export function getDescriptorComplete(descriptorData: {
-  url: string,
-  descriptor?: Descriptor
-}) {
-  return {
-    type: GET_DESCRIPTOR_COMPLETE,
-    payload: descriptorData
-  }
-}
+export const getChildrenComplete = /*#__PURE__*/ createAction<{ url: string, children?: Array<Item> }>('GET_CHILDREN_COMPLETE');
 
-export function getChildren(url: string) {
-  return {
-    type: GET_CHILDREN,
-    payload: url
-  }
-}
+export const getTree = /*#__PURE__*/ createAction<{ url: string, depth?: Number }>('GET_TREE');
 
-export function getChildrenComplete(childrenData: {
-  url: string,
-  children?: Array<Item>,
-}) {
-  return {
-    type: GET_CHILDREN_COMPLETE,
-    payload: childrenData
-  }
-}
+export const getTreeComplete = /*#__PURE__*/ createAction<{ url: string, tree?: Item }>('GET_TREE_COMPLETE');
 
-export function getTree(url: string);
-export function getTree(url: string, depth: Number);
-export function getTree(url: string, depth: Number = 1) {
-  return {
-    type: GET_TREE,
-    payload: {
-      url,
-      depth
-    }
-  }
-}
+export const getNav = /*#__PURE__*/ createAction<{ url: string, depth?: Number, currentPageUrl?: string }>('GET_NAV');
```

Comment:
@rart A couple of actions like this one changed because they were accepting multiple parameters. With @redux/toolkit, the payload becomes an object (the defaults are already handled in the services from @craftercms/content. README file and CHANGELOG are already updated with this change.